tests/robot/suites/traffic/veth_afpacket_memif_vxlan_traffic/veth_afpacket_memif_vxlan_traffic_other_vrf.robot

```diff
@@ -0,0 +1,166 @@
+#Library      SSHLibrary      timeout=60s
+#Library      String
+
+Resource     ../../../variables/${VARIABLES}_variables.robot
+Resource     ../../../libraries/all_libs.robot
+Resource    ../../../libraries/pretty_keywords.robot
+
+Suite Setup       Testsuite Setup
+Suite Teardown    Testsuite Teardown
+
+*** Variables ***
+${VARIABLES}=          common
+${ENV}=                common
+${CONFIG_SLEEP}=       1s
+${RESYNC_SLEEP}=       1s
+# wait for resync vpps after restart
+${RESYNC_WAIT}=        30s
+
+*** Test Cases ***
+Configure Environment
+    [Tags]    setup
+    Add Agent VPP Node    agent_vpp_1
+    Add Agent VPP Node    agent_vpp_2
+
+Show Interfaces Before Setup
+    vpp_term: Show Interfaces    agent_vpp_1
+    vpp_term: Show Interfaces    agent_vpp_2
+    Write To Machine    vpp_agent_ctl    vpp-agent-ctl ${AGENT_VPP_ETCD_CONF_PATH} -ps
+
+Setup Interfaces
+    vpp_ctl: Put Memif Interface With IP    node=agent_vpp_1    name=vpp1_memif1    mac=62:61:61:61:61:61    master=true    id=1    ip=192.168.1.1    vrf=1
+    vpp_ctl: Put Veth Interface With IP    node=agent_vpp_1    name=vpp1_veth1    mac=12:11:11:11:11:11    peer=vpp1_veth2    ip=10.10.1.1    vrf=1
+    vpp_ctl: Put Veth Interface    node=agent_vpp_1    name=vpp1_veth2    mac=12:12:12:12:12:12    peer=vpp1_veth1
+    vpp_ctl: Put Afpacket Interface    node=agent_vpp_1    name=vpp1_afpacket1    mac=a2:a1:a1:a1:a1:a1    host_int=vpp1_veth2    vrf=1
+    vpp_ctl: Put VXLan Interface    node=agent_vpp_1    name=vpp1_vxlan1    src=192.168.1.1    dst=192.168.1.2    vni=5    vrf=1
+    # # temporarily START to replace bad implemented vrf for vxlan
+    # ${int3}=    vpp_ctl: Get Interface Internal Name    agent_vpp_1    vpp1_vxlan1
+    # vpp_term: Issue Command    node=agent_vpp_1    command=create vxlan tunnel src 192.168.1.1 dst 192.168.1.2 vni 5 del
+    # vpp_term: Issue Command    node=agent_vpp_1    command=create vxlan tunnel src 192.168.1.1 dst 192.168.1.2 vni 5 encap-vrf-id 1 decap-next l2
+    # sleep    2
+    # # temporarily END
+    @{ints}=    Create List    vpp1_vxlan1    vpp1_afpacket1
+    vpp_ctl: Put Bridge Domain    node=agent_vpp_1    name=vpp1_bd1    ints=${ints}
+
+    vpp_ctl: Put Memif Interface With IP    node=agent_vpp_2    name=vpp2_memif1    mac=62:62:62:62:62:62    master=false    id=1    ip=192.168.1.2
+    vpp_ctl: Put Veth Interface With IP    node=agent_vpp_2    name=vpp2_veth1    mac=22:21:21:21:21:21    peer=vpp2_veth2    ip=10.10.1.2
+    vpp_ctl: Put Veth Interface    node=agent_vpp_2    name=vpp2_veth2    mac=22:22:22:22:22:22    peer=vpp2_veth1
+    vpp_ctl: Put Afpacket Interface    node=agent_vpp_2    name=vpp2_afpacket1    mac=a2:a2:a2:a2:a2:a2    host_int=vpp2_veth2
+    vpp_ctl: Put VXLan Interface    node=agent_vpp_2    name=vpp2_vxlan1    src=192.168.1.2    dst=192.168.1.1    vni=5
+    @{ints}=    Create List    vpp2_vxlan1    vpp2_afpacket1
+    vpp_ctl: Put Bridge Domain    node=agent_vpp_2    name=vpp2_bd1    ints=${ints}
+
```

Comment:
always use sleep for 10s (sync time) after configuration and before checking status